tests/functional/i/iterating_modified.py

```diff
@@ -0,0 +1,41 @@
+"""Tests for iterating-modified messages"""
+# pylint: disable=locally-disabled,not-callable,unnecessary-comprehension,suppressed-message,useless-suppression
```

Comment:
```suggestion
# pylint: disable=ot-callable, unnecessary-comprehension,
```

Reply:
when I run pytest locally it says it's necessary...